frontend/src/app/features/api-docs/swagger-ui/swagger-ui.component.html

```diff
@@ -0,0 +1,3 @@
+<div id="docs-wrapper">
```

Comment:
Do you need the wrapper? I think you might even be able to have no template at all, and instead do something like

```
@Component({
  // other stuff
  // ...
  host: {
    id: 'swagger',
  },
})
```

Which would reduce both the complexity of the code but most importantly of the resulting html

Reply:
Tried this but it doesn't seem to work. The compiler complains about a missing template. Leaving it as it is in that case.